src/leaflike/bookmarks/db.clj

```diff
@@ -1,17 +1,19 @@
 (ns leaflike.bookmarks.db
   (:require [clojure.java.jdbc :as jdbc]
-            [leaflike.config :refer [db-spec]]
-            [honeysql.core :as sql]
-            [honeysql.helpers :as helpers]
-            [honeysql.types :as types]))
+            [honeysql.core     :as sql]
+            [honeysql.helpers  :as helpers]
+            [honeysql.types    :as types]
+            [clojure.string    :as str]
+            [leaflike.config   :refer [db-spec]]))
 
 (defn format-tags
   [{:keys [tags] :as params}]
   ;; check if tag exists
   ;; check if tags is nil?
   (if (nil? tags)
     params
-    (update params :tags #(types/array %))))
+    (do (str/split tags #"(\w)(\s+)([\.,])")
```

Comment:
The result of this `split` isn't being used anywhere. `tags` is still the unsplit string.

Also, why isn't `(str/split tags #",")` enough?